### ISSUE TITLE
The package in the registry unexpectedly contains optionalDependencies

### ISSUE THREAD

**Issue by lukyth** [piotrwitek/typesafe-actions#118]
## Description

The latest version (3.1.0) of this package in the Yarn registry (https://registry.yarnpkg.com/typesafe-actions/-/typesafe-actions-3.1.0.tgz) contains an `optionalDependencies` as well.
```json
{
  "optionalDependencies": {
    "typescript": "3.3.3"
  }
}
```

## Steps to Reproduce

1. Download the package directly from https://registry.yarnpkg.com/typesafe-actions/-/typesafe-actions-3.1.0.tgz or run `yarn add typesafe-actions` and checkout the downloaded package in `node_modules`.
2. Checkout `package.json` in the package.

## CodeSandbox to help you reproduce issue in isolation
> Please use this sandbox as an isolated environment to reproduce the bug: https://codesandbox.io/s/github/piotrwitek/typesafe-actions/tree/master/codesandbox

## Expected behavior

`package.json` in the `master` branch (https://github.com/piotrwitek/typesafe-actions/blob/master/package.json#L55) doesn't have `optionalDependencies`. The published one should be the same.

## Suggested solution(s)

Not sure how did this happen. Re-publish the package might help?

## Project Dependencies
- Typesafe-Actions Version: 3.1.0
- TypeScript Version: 3.3.3333
- tsconfig.json:
```json
{
  "compilerOptions": {
    /* Basic Options */
    "target": "es5",
    "module": "commonjs",
    "lib": ["es2017", "dom"],
    "jsx": "react",
    "sourceMap": true,
    "removeComments": true,
    "noEmit": true,

    /* Strict Type-Checking Options */
    "strict": true,

    /* Additional Checks */
    "noUnusedLocals": true,
    "noUnusedParameters": true,
    "noImplicitReturns": true,
    "noFallthroughCasesInSwitch": true,

    /* Module Resolution Options */
    "moduleResolution": "node",
    "typeRoots": ["./node_modules/@types", "./typings"],
    "esModuleInterop": true,

    /* Experimental Options */
    "experimentalDecorators": true,

    /* Advanced Options */
    "skipLibCheck": true,
    "forceConsistentCasingInFileNames": true
  },
  "exclude": ["dist", ".cache", "node_modules"]
}
```

## Environment
- Browser and Version: Google Chrome 72.0.3626.109
- OS: macOS Mojave 10.14.3
- Node Version: 10.13.0
- Package Manager and Version: Yarn 1.13.0

**Comment by piotrwitek** [comment]
@lukyth thanks for the report
It's not a bug, TS was optional dep in v3.1.0, I removed it recently and it'll be release with typesafe-actions@3.1.1.

Is there any specific issue that you're experiencing right now because of this?

**Comment by lukyth** [comment]
It doesn't cause any project issue. I just notice that there're two TypeScript versions in my dependencies and it bugs me 😅 So I investigated until I found this.

Thank you for taking care of this issue though! 😄

**Comment by piotrwitek** [comment]
Closed by ca8468cfe2f9935f59901381bdc02cf67d0526f6
